woocommerce/woocommerce-gutenberg-products-block
# Move payment-methods folder from base components to cart-checkout block

Part of #1962.

This PR moves the  `payment-methods` folder, as discussed in #1962. It doesn't close the issue completely, as there are other folders to be moved. 

What is done in this PR:
- `/base/components/payment-methods` is now under `blocks/cart-checkout`

### How to test the changes in this Pull Request:

1. Verify the new folder structure and whether it makes sense.
2. Make sure the project still builds and do some smoke testing to verify _Cart_, _Checkout_ and blocks continue working.
